test/chpldoc/linkage-spec/exportTest.doc.good

```diff
@@ -72,6 +72,10 @@ exportTest.doc
 
 .. function:: export proc allThreeCommentless2(val: int): int
 
+.. function:: export proc withExternCName()
+
+.. function:: export proc withParamExternCName()
```

Comment:
It might be nice to open an issue for having the C name displayed as well in a nice way, but this is a reasonable update for this PR

Reply:
#10387

Reply:
Thanks :)